app/views/devise/passwords/edit.html.slim

```diff
@@ -14,6 +14,9 @@
         = f.input :reset_password_token, as: :hidden
         = f.full_error :reset_password_token
         = f.input :password, label: 'New password', required: true, autofocus: true
+        = render 'devise/shared/password_strength'
         = f.input :password_confirmation, label: 'Confirm your new password', required: true
         = f.button :submit, 'Change my password'
       = render 'devise/shared/links'
+
+== javascript_include_tag 'misc/pw-strength'
```

Comment:
Derp! Yes, you are right. I didn't realize this was doing a `script src`. However, I think the whole point of the asset pipeline is that everything is minified into a single file, so that the browser only makes one HTTP request, as opposed to 3 different ones each time this page is accessed: one for `application.js`, one for `app/utils.js`, and one for `misc/pw-strength.js`.